columnflow/tasks/cutflow.py

```diff
@@ -338,7 +342,7 @@ def run(self):
                     }]
 
                     # axis reductions
-                    h = h[{"process": sum, "category": sum, "mc_weight": sum}]
+                    h = h[{"process": sum, "category": sum, "event": sum}]
```

Comment:
Does the sum over `event` include potential `flow` entries? Likely only a problem if the range of the `event` `od.Variable` is too small (see my other comment on the binning).

Reply:
Great, thanks for checking! I also think it's probably better to include `flow` bins in `sum` reductions, but it's good to be aware of this behavior in case this is ever not wanted.